src/main/scala/io/citrine/lolo/bags/BaggedPrediction.scala

```diff
@@ -231,37 +196,45 @@ case class MultiPointBaggedPrediction(
     )
   }
 
-  override def getImportanceScores(): Option[Seq[Seq[Double]]] = Some(scores)
+  override def importanceScores: Option[Seq[Seq[Double]]] = Some(scores)
 
-  override def numPredictions: Int = expectedMatrix.length
+  override lazy val stdDevObs: Option[Seq[Double]] = Option.when(!disableBootstrap)(varObs.map(math.sqrt))
+
+  override lazy val stdDevMean: Option[Seq[Double]] = Option
+    .when(disableBootstrap) {
+      // If bootstrap is disabled, rescale is unity and treeVariance is our only option for UQ.
+      // Since it's not recalibrated, it's best considered to be a confidence interval of the underlying weak learner.
+      assert(rescaleRatio == 1.0)
+      varObs.map(math.sqrt)
+    }
+    .orElse {
+      val std = variance(rawExpected.toVector, expectedMatrix, NibJMat, NibIJMat).map(math.sqrt)
+      Some(std)
+    }
 
   /* transpose to be (# training) x (# models) */
-  lazy val Nib: Vector[Vector[Int]] = NibIn.transpose
+  private lazy val Nib: Vector[Vector[Int]] = NibIn.transpose
```

Comment:
These helper variables are all `private` on the above `SinglePointBaggedPrediction`, so making private here for consistency.